src/offer/offerRepository.js

```diff
@@ -81,67 +79,70 @@ export class OfferRepository {
       const idsForQuery = ids.map(() => UUID_TO_BIN).join(', ');
 
       const offerItemsQuery = `
-      SELECT 
-        BIN_TO_UUID(o.id) AS offerId,
-        oi.item_Name AS offerItemName,
-        oi.Quantity AS offerQuantity,
-        i.Img AS offerItemImg
-      FROM 
-        offers o
-        LEFT JOIN offer_items oi ON o.id = oi.offer_id
-        LEFT JOIN items i ON oi.item_Name = i.Name
-      WHERE 
-        o.id IN (${idsForQuery})
+    SELECT 
+    BIN_TO_UUID(o.id) AS offerId,
+    oi.item_Name AS offerItemName,
+    oi.Quantity AS offerQuantity,
+    i.Img AS offerItemImg
+    FROM 
+    offers o
+    LEFT JOIN offer_items oi ON o.id = oi.offer_id
+    LEFT JOIN items i ON oi.item_Name = i.Name
+    WHERE 
+    o.id IN (${idsForQuery})
     `;
       const offerItems = await this.mySQLConnection.executeQuery(offerItemsQuery, ids);
-
       const requestItemsQuery = `
-      SELECT 
+    SELECT 
         BIN_TO_UUID(o.id) AS offerId,
         ri.item_Name AS requestItemName,
         ri.Quantity AS requestQuantity,
         ir.Img AS requestItemImg
-      FROM 
+        FROM 
         offers o
         LEFT JOIN request_items ri ON o.id = ri.offer_id
         LEFT JOIN items ir ON ri.item_Name = ir.Name
-      WHERE 
+        WHERE 
         o.id IN (${idsForQuery})
-    `;
+        `;
```

Comment:
The styles in the written SQL are not standard. See the option to configure the linter to leave it standard or manually unify the styles